records_mover/db/unloader.py

```diff
@@ -49,6 +51,12 @@ def can_unload_to_scheme(self, scheme: str) -> bool:
         this may depend on whether a temporary location has been configured."""
         ...
 
+    @abstractmethod
+    @contextmanager
+    def temporary_unloadable_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
+        # TODO document
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker